content/tutorials/internals/howbrowserswork/pt/index.html

```diff
@@ -797,7 +797,7 @@ <h4 id="Browsers_error_tolerance">Tolerância a erros dos navegadores</h4>
 Novamente, o comentário é auto-explicativo.
 
 <blockquote>
-Suporte para html realmente corrompido. Nunca fechamos a tag body, já que algumas páginas da web estúpidas a fecham antes do final efetivo do documento. Vamos confiar na chamada end() para o fechamento.
+Suporte para html realmente corrompido. Nunca fechamos a tag body, já que algumas páginas estúpidas da web  a fecham antes do final efetivo do documento. Vamos confiar na chamada end() para o fechamento.
```

Comment:
-- pt_BR:
@ewertonorg , obrigado pelo PR. Você pode também remover o espaço duplo que ficou sobrando depois de  "estúpidas da web"?

-- en_US:
@ewertonorg , thanks for the PR. Could you also remove the extra space after "estúpidas da web"?